textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/dictionarymatcher/Dictionary.java

```diff
@@ -0,0 +1,33 @@
+package edu.uci.ics.textdb.dataflow.dictionarymatcher;
+
+import java.util.List;
+
+import edu.uci.ics.textdb.api.common.IDictionary;
+
+/**
+ * @author Sudeep [inkudo]
+ *
+ */
+public class Dictionary implements IDictionary {
+
+    private int cursor = -1;
+    private List<String> dict;
+
+    public Dictionary(List<String> dict) {
```

Comment:
Changed the instance variable and the parameter in the constructor from dict to stringList.